metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/DataProcessAspect.pdl

```diff
@@ -2,12 +2,30 @@ namespace com.linkedin.metadata.aspect
 
 import com.linkedin.metadata.key.DataProcessKey
 import com.linkedin.dataprocess.DataProcessInfo
+import com.linkedin.dataprocess.DataProcessInputOutput
+import com.linkedin.dataprocess.EditableDataProcessProperties
 import com.linkedin.common.Ownership
 import com.linkedin.common.Status
+import com.linkedin.common.GlobalTags
+import com.linkedin.common.BrowsePaths
+import com.linkedin.common.GlossaryTerms
+import com.linkedin.common.InstitutionalMemory
 import com.linkedin.common.DataPlatformInstance
 
 /**
  * A union of all supported metadata aspects for a data process
  */
 @deprecated = "Use DataJob instead."
-typeref DataProcessAspect = union[DataProcessKey, Ownership, DataProcessInfo, Status]
+typeref DataProcessAspect = union[
```

Comment:
Since we don't want to make the MCE (snapshot) schema any bigger, let's omit these changes, and just add the aspects to `entity-registry.yml` in `metadata-models/src/main/resources`

Reply:
addressed in https://github.com/linkedin/datahub/pull/4114/commits/905e25383114e9d1315ed8a66041fbc4acb10b8d

do you want me to remove updates on the `DataProcessAspect.pdl`? if so, is there any issue on having this out-of-sync with the registry?

or instead do you want me to delete the whole `DataProcessAspect.pdl`?